flexget/plugins/filter/movie_queue.py

```diff
@@ -347,15 +361,15 @@ def queue_del(title=None, imdb_id=None, tmdb_id=None, session=None, movie_id=Non
         return title
     except NoResultFound as e:
         raise QueueError(
-                'title=%s, imdb_id=%s, tmdb_id=%s, movie_id=%s not found in queue' % (
-                    title, imdb_id, tmdb_id, movie_id))
+            'title=%s, imdb_id=%s, tmdb_id=%s, movie_id=%s not found in queue' % (
+                title, imdb_id, tmdb_id, movie_id))
     except MultipleResultsFound:
         raise QueueError('title=%s, imdb_id=%s, tmdb_id=%s, movie_id=%s matches multiple results in queue' %
                          (title, imdb_id, tmdb_id, movie_id))
 
 
 @with_session
-def queue_forget(title=None, imdb_id=None, tmdb_id=None, session=None, movie_id=None):
+def queue_forget(title=None, imdb_id=None, tmdb_id=None, session=None, movie_id=None, queue_name=None):
```

Comment:
We could. Isn't that considered bad practice though? I thought that was why we set quality to any internally as well